docs/custom-commands/roles.md

```diff
@@ -0,0 +1,14 @@
+---
+id: command-roles
+sidebar_position: 4
+---
+
+# Roles
+
+Roles allow you to set who has access to each command. This can be set to everyone, non-subscriber, subscriber, VIP, Founder, Moderator, and Broadcaster but also can be used with custom roles. 
+
+:::info
+When no roles are selected, the command works for all users.
```

Comment:
Not sure if this is the best phrasing, maybe like ```If no roles are selected, it's triggerable by all users``` or something? I like the idea of saying "triggering commands" as a standard across all docs imo